server/app/db/schemas/bot_types.py

```diff
@@ -0,0 +1,34 @@
+"""Contains all the schemas for data validation of botType CRUD functions
+_"""
+
+from typing import Any, Tuple
+
+from pydantic import BaseModel
+
+from .timestamp import TimestampInDBPlugin
+
+
+class BotTypeBase(BaseModel):
+    name: str
+
+
+class InsertBotType(BotTypeBase):
+    pass
+
+
+class BotTypeInDB(BotTypeBase, TimestampInDBPlugin):
+    id: int
+
+    class Config:
+        orm_mode = True
+
+
+def create_BotTypeInDB_from_tuple(data: Tuple[Any, ...]) -> BotTypeInDB:
+    """Utility function which creates a ButTypeInDB model from the database row."""
```

Comment:
fix typo BotTypeInDB

Reply:
typo in the sense you want me to convert BotTypeInDB to snake case ?

Reply:
Utility function which creates a ~ButTypeInDB~ BotTypeInDB...